examples/org.osate.examples/examples/safety-tutorial/packages/GPSSystem.aadl

```diff
@@ -164,20 +164,20 @@ public
 	-- Now with the GPSProcessing specification that includes the redundancy logic for the sensor inputs.
 	system implementation GPS.BasicRedundancy extends GPS.Basic
 		subcomponents
-			processing: refined to process GPSParts::GPSProcessing_redundancy;
+			processing: refined to process GPSParts::GPSProcessing_redundancy{Classifier_Substitution_Rule => Type_Extension;};
 	end GPS.BasicRedundancy;
 
 	-- here with the GPSProcessing specification that includes the redundancy logic for the sensor inputs via an error state.
 	-- Should produce the same fault tree as BasicRedundancy
 	system implementation GPS.BasicState extends GPS.basic
 		subcomponents
-			processing: refined to process GPSParts::GPSProcessing_redundancyState;
+			processing: refined to process GPSParts::GPSProcessing_redundancyState{Classifier_Substitution_Rule => Signature_Match;};
```

Comment:
This results in an info message that "Signature_Match" isn't implemented in OSATE. Can the example be changed such that it doesn't contain unsupported things?

Reply:
I discussed this with Anna. This info message is better than the warning before. Furthermore, this is the expected fix to match the model semantics.